README.rst

```diff
@@ -1,76 +1,62 @@
-=======
-Astropy
-=======
+|Astropy Logo|
+----
 
 .. container::
 
     |Actions Status| |CircleCI Status| |Coverage Status| |PyPI Status| |Documentation Status| |Pre-Commit| |Ruff| |Zenodo|
 
-The Astropy Project (https://astropy.org/) is a community effort to develop a
-single core package for Astronomy in Python and foster interoperability between
-Python astronomy packages. This repository contains the core package which is
-intended to contain much of the core functionality and some common tools needed
-for performing astronomy and astrophysics with Python.
+----
 
-Table of Contents
-=================
+|User Stats|
```

Comment:
Thanks.

I am not sure about having this right at the top. If I were someone new wondering what Astropy is, these stats don't mean anything to me, I would want to read "about" first.

Isn't this more appropriate under "contributing"?

Reply:
My thought was that the 'user stats image' is best near the top, as it shows aspects of the status of the repo, similar to the badges. The metrics it lists are general I'd say - basic git things (contributor, issue, PR counts) and citations (which probably most people who use astropy are familiar with). And since it's kind of a big image, I think it would break up the readme and make it less readable if moved to a lower section.

Reply:
For `README.rst`, I agree with you that in a way it belongs in the line of badges. Indeed, one might want to put it *above* the badges... (but for the main docs page, I feel differently)

Reply:
Great, I moved the image above the badges in `README.rst` and to the bottom of the main docs page in [6a6dadd](https://github.com/astropy/astropy/pull/16822/commits/6a6dadd3c4b616ec01b62b443eb3ce7f9943c9a8).